pages/api/payment-received.ts

```diff
@@ -62,30 +62,36 @@ const paymentReceivedHandler: NextApiHandler = async (req, res) => {
       where: { id: applicationId },
       select: { email: true, firstName: true },
     });
-    const applicationEmail = application?.email;
-    const applicationFirstName = application?.firstName;
 
-    // Update application
-    await prisma.application.update({
-      where: { id: applicationId },
-      data: {
-        shopifyPaymentStatus: ShopifyPaymentStatus.RECEIVED,
-        shopifyConfirmationNumber: `${shopifyOrderId}`,
-        paidThroughShopify: true,
-        donationAmount: donationAmount,
-      },
-    });
+    if (!application) {
+      // Bad request, occurs if application ID does not exist
+      res.status(400).end();
```

Comment:
We should return an error message with the 400 request so that we know the reason for failure

Reply:
Done! Lmk if you had a different error message in mind, currently have `application with given id does not exist` as the error message